lessons/lesson3.md

```diff
@@ -0,0 +1,150 @@
+## Lesson 3: Fire Event using Event SDK
+
+### Fire Event
```

Comment:
With the latest app template, the majority of content here is not relevant anymore.
My proposal is to go through the `aio app init` route with the latest CLI, and update the content accordingly.

Reply:
@duynguyen it is still relevant, the template just the sample code `actions/publish-event/index.js` as the sample code to provide the way convert an input json to cloud event and publish to events, but still user need to fill to customized the custom event

Reply:
but I add the template here

Reply:
well, AFAICS the code from template `actions/publish-event/index.js` would do everything as shown in the custom code in `actions/event/index.js`, right? For the users it's better to just show how to use the `publish-event` action based on the template, mapping the value of org ID, API key, event code, etc. then everything works OOTB.